packs/rage-of-elements-bestiary/ifrit-shuyookh.json

```diff
@@ -1501,11 +1501,51 @@
             },
             "type": "action"
         },
+        {
+            "_id": "QziuCiW6YG0yD2jV",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Exploit Regret",
+            "sort": 1800000,
+            "system": {
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "description": {
+                    "value": "<p><strong>Trigger</strong> A creature fails an attack against the shuyookh</p>\n<hr />\n<p><strong>Effect</strong> The shuyookh asks the triggering creature if it wishes it had hit. If the creature indicates yes, the shuyookh takes damage as if the attack had hit and the expression of regret makes the shuyookh @UUID[Compendium.pf2e.conditionitems.Item.Quickened] for [[/gmr 1d4 #Quickened Rounds]]{1d4 rounds}; they can use their extra action to Fly, Stride, or Strike.</p>"
```

Comment:
I don't believe we use /gmr anywhere else - we typically use /br

Reply:
gmr is more appropriate here since it's the GM rolling